elements/lisk-validator/src/formatter.ts

```diff
@@ -17,7 +17,9 @@ import * as Ajv from 'ajv';
 // tslint:disable-next-line no-require-imports
 import addKeywords = require('ajv-merge-patch');
```

Comment:
This is very complicated stuff, dynamically modifying the JSON-schema runtime and then validate data against that particular in-memory schema. We should avoid using it, unless we have a particular use-case which can't be handled otherwise. Instead we should always recommend to use a statically defined JSON-Schema.

Reply:
I think this is outside the scope of this PR for which only methods that were missing were added to this library. Please open an issue with suggestions for lisk-validator

Reply:
If you agree to feedback and think its out of scope, then consolidate all such feedback in an issue and resolve them with issue reference.

Reply:
Removed. Some tests of validator were using but I changed them to `allOf`